geode-core/src/test/java/org/apache/geode/internal/cache/versions/RegionVersionVectorTest.java

```diff
@@ -61,6 +61,49 @@
   @Rule
   public ExpectedException expectedException = ExpectedException.none();
 
+  /**
+   * server1 will simulate doing a sync with another server for operations performed
+   * by server2. server3 is another server in the cluster that we don't care about
+   */
+  @Test
+  public void testSynchronizationVectorContainsAllVersionsForOwnerAndNonTarget() {
+    final String local = NetworkUtils.getIPLiteral();
+    InternalDistributedMember server1 = new InternalDistributedMember(local, 101);
+    InternalDistributedMember server2 = new InternalDistributedMember(local, 102);
+    InternalDistributedMember server3 = new InternalDistributedMember(local, 103);
+
+    RegionVersionVector rv1 = new VMRegionVersionVector(server1);
+    rv1.updateLocalVersion(10);
+    rv1.recordVersion(server2, 1);
+    rv1.recordVersion(server2, 5);
+    rv1.recordVersion(server2, 8);
+    rv1.recordVersion(server3, 1);
+    rv1.recordVersion(server3, 3);
+    RegionVersionVector singletonRVV = rv1.getCloneForTransmission(server2);
+    assertTrue(singletonRVV.isForSynchronization());
+    assertEquals(singletonRVV.getOwnerId(), server1);
+    assertTrue(singletonRVV.getMemberToVersion().containsKey(server2));
+    assertFalse(singletonRVV.getMemberToVersion().containsKey(server3));
```

Comment:
if 
  assertFalse(singletonRVV.getMemberToVersion().containsKey(server3));
how can
  assertTrue(singletonRVV.contains(server3, 1));
?

Reply:
The first assertion ensures that we don't have a version holder for server3, who owns the RVV.  The second assertion ensures that the RVV returns _true_ for any contains() concerning the owner of the RVV.  That is what this change set fixes.